### ISSUE TITLE
Add Azure Pipeline and add webhook to github

### ISSUE THREAD

**Issue by davidzwa** [marshmallow-code/django-rest-marshmallow#139]
@sloria I can help set up Azure Pipelines. Do you have a reference pipeline file for travis I can take a look at?

On the pipeline matter:
- What/where is the python matrix/strategy for TravisCI you currently use?
- Are you using pytest?

On administration:
- What is your Azure account I can use to invite you as admin to the project?
- How do I make sure the DevOps falls under your org/name with you as admin? I have created https://dev.azure.com/marshmallow-code/django-rest-marshmallow by following https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml
- Can I PR an azure-pipeline.yml later, so that you can setup the github action/hook side of things?
- I need you to authorize the Azure project to use the github webhook. If I do it, it goes to my github account only, which I want to avoid.
![image](https://user-images.githubusercontent.com/6005355/79013030-0b065100-7b68-11ea-9441-699e3ebd70c8.png)

**Comment by sloria** [comment]
Thanks for offering to take this on. Take a look at the other repos in marshallow-code for an example. I'll still need to authorize AP on this repo, but you can still send a PR adding the config file

**Comment by davidzwa** [comment]
Hmm, I see. If you give me your Azure account, I can let you authorize and PR the `azure-pipelines.yml` file with one click. The work is in the debugging step of the pipeline I presume, so I don't think just adding the `.yml` like [your azure-pipelines.yml example](https://github.com/marshmallow-code/marshmallow/blob/dev/azure-pipelines.yml) really bears a lot of meaning if I cant test it.

I'll wait for you to send me your Azure account name, so you can be invited and authorizing the org / take over permissions.

**Comment by sloria** [comment]
I've just added the app to this repo and added it under the same Azure project that has all the other marshmallow libs. I think if you send a PR adding the azure-pipelines.yml it should trigger a build.. maybe...?

**Comment by davidzwa** [comment]
If the webhook is setup it should! I think that still needs to be done first. The marshmallow DRF repo needs to be known to azure for anything to trigger. 
Please click 'add pipeline' > 'github repos' > add python azure-pipelines.yml > make PR.

I can then start looking at the PR when it's here on github.

**Comment by sloria** [comment]
ok, here's the PR generated through AP: https://github.com/marshmallow-code/django-rest-marshmallow/pull/140/files

**Comment by davidzwa** [comment]
Perfect! I will have to make another PR, as I need to fork, but now the webhook should work. Well done!

**Comment by davidzwa** [comment]
The pipeline works! @sloria next step is to make it a requirement for pull-requests once your content with PR #141

**Comment by davidzwa** [comment]
See PR for info about status, as I need a bit of feedback.
